### ISSUE TITLE
Insufficient/non-intuitive documentation for basic tasks

### ISSUE THREAD

**Issue by Code7R** [rrthomas/pdfjam#30]
Hi,

I wanted to use pdfjoin after a while of not having to, and run into some obstacles. So how do I do it? Not clear. `pdfjoin` is mentioned at some location but is now considered replaced. By what? Not clear. 
Tried the manpage? Worthless, refers to program help only. Ok, where?

`pdfjam -h | grep join`

-> NO information. How do I join them? This should be mentioned in the help.

So, trying a blind guess:

pdfjam input_*.pdf

Resulted in a PDF file, yes. But I found out that I need to rotate one of the inputs. How to do it?

`pdfjam -h | less`

There is no --rotate option or similar. Just something mentioning a suffix -rotate.

**Comment by lemniscati** [comment]
You can still use the script `pdfjoin`:
https://github.com/DavidFirth/pdfjam-extras/blob/master/bin/pdfjoin .

But, wrapper scripts are no longer included in `pdfjam` (cf. [README.md][1]).

[1]: https://github.com/DavidFirth/pdfjam/blob/9664dd92ea9385f9e60a397561ec238305fcea92/README.md#wrappers

**Comment by DavidFirth** [comment]
Many thanks @leminscati for the pointer to where pdfjoin can still be found if wanted.  Note that in my view `pdfjoin` is redundant, though: just use `pdfjam` directly, with suitable arguments.

@Code7R did not say which version of pdfjam they were commenting on.  Let's assume it was the latest version.  In that case, the very last line of output in `pdfjam -h` should tell where to look for more information.  I hope that works.

Thank you @Code7R for telling me that the man page is worthless to you.  Please feel free to delete it from your system.  I will happily refund all that you have paid me for it!

I am closing this now because I do not think there is anything to fix.  If anyone disagrees, then I would of course welcome any helpful patches for the existing documentation.  Thanks again!

**Comment by Code7R** [comment]
Closing? You think because you are unable to see the problem then it does not exist?
I told what the problem is. Neither the manpage nor the help output do tell the user how to do the basic tasks like
"join" or "rotate" or "flip".
And please don't play the "did you pay me" game on me. It's like in LIfe of Brian (what did Romans bring us...). I am FOSS author and integrator and in case you forgot, I have introduced pdfjam to Debian back in 2004. And now you want ME to improve the documentation by finding out the option syntax which is buried somewhere in the shell code? Instead of the authors who know them already? Forget it, I have better things to do.

Oh, and:
  pdfjam: This is pdfjam version 3.03.

**Comment by DavidFirth** [comment]
> Neither the manpage nor the help output do tell the user how to do the basic tasks like "join" or "rotate" or "flip".

True.  Lots of other things are not mentioned there, either.  This is deliberate.  The man page directs to `pdfjam -h`, which gives formal documentation for pdfjam, and also a link to the README page which provides more documentation still --- and on that README page you'll find words like join, flip, rotate and plenty of others.

Hence closed, still.  I sense that we won't see eye to eye on this --- sorry about that.

**Comment by rrthomas** [comment]
The man page can be improved with `help2man`.

**Comment by mgkurtz** [comment]
I will see to it, that I use the words “join” and “rotate” at the appropriate places when I update the documentation as planned in https://github.com/pdfjam/pdfjam/issues/34#issuecomment-2621962173.

> I found out that I need to rotate one of the inputs. How to do it?

For rotating just one input, the most simple solution currently is to process this file first and then jam the rotated file together with the rest. I may put such an example in the documentation as well.
